deployment/grid/terraform/vpc/vpc.tf

```diff
@@ -157,3 +157,13 @@ resource "aws_security_group_rule" "https" {
   cidr_blocks       = [data.aws_vpc.selected.cidr_block]
   security_group_id = module.vpc.default_security_group_id
 }
+
+resource "aws_security_group_rule" "egress_rule" {
```

Comment:
Did we ever confirm that this was a one off issue or does it happen all the time? Need to confirm that we don't get breaks if this rule already exists.